### ISSUE TITLE
Histogram display using masks > 2D 

### ISSUE THREAD

**Issue by agahkarakuzu** [qMRLab/qMRLab#259]
Ran into this error message while trying to see Histogram for `denoise_mppca`: 

```
Attempt to grow array along ambiguous dimension.

Error in qMRLab>Histogram_Callback (line 599)
        Map(~rot90(Data.(Model).Mask)) = 0;

Error in gui_mainfcn (line 95)
        feval(varargin{:});

Error in qMRLab (line 37)
    gui_mainfcn(gui_State, varargin{:});

Error in
matlab.graphics.internal.figfile.FigFile/read>@(hObject,eventdata)qMRLab('Histogram_Callback',hObject,eventdata,guidata(hObject)) 
Error while evaluating UIControl Callback.
```

**Comment by tanguyduval** [comment]
note that there might be some change in the use of the histogram function if we plan to use the viewer imtool3D #232

**Comment by tanguyduval** [comment]
solved in  #296
